azurerm/resource_arm_autoscale_setting_test.go

```diff
@@ -0,0 +1,831 @@
+      } 
+    
+      os_profile_linux_config { 
+        disable_password_authentication = false 
+      } 
+    
+      network_profile { 
+        name    = "terraformnetworkprofile" 
+        primary = true 
+    
+        ip_configuration { 
+          name                                   = "TestIPConfiguration" 
+          subnet_id                              = "${azurerm_subnet.test.id}" 
+          load_balancer_backend_address_pool_ids = ["${azurerm_lb_backend_address_pool.bpepool.id}"] 
+          load_balancer_inbound_nat_rules_ids    = ["${element(azurerm_lb_nat_pool.lbnatpool.*.id, count.index)}"] 
+        } 
+      } 
+    
+      tags { 
+        environment = "staging" 
+      } 
+    } 
+
+    resource "azurerm_autoscale_setting" "test" {
+      name                = "myAutoscaleSetting"
+      enabled             = true
+      resource_group_name = "${azurerm_resource_group.test.name}"
+      location            = "${azurerm_resource_group.test.location}"
+      target_resource_uri = "${azurerm_virtual_machine_scale_set.test.id}"
+      profile {
+        name = "defaultProfile"
+        capacity {
+          default = 1
+          minimum = 1
+          maximum = 10
+        }
+        rule {
+          "metric_trigger" {
+            "metric_name"         = "Percentage CPU"
+            "metric_resource_uri" = "${azurerm_virtual_machine_scale_set.test.id}"
+            "time_grain"          = "PT1M"
+            "statistic"           = "Average"
+            "time_window"         = "PT5M"
+            "time_aggregation"    = "Average"
+            "operator"            = "GreaterThan"
+            "threshold"           = 75
+          }
+          "scale_action" {
+            "direction" = "Increase"
+            "type"      = "ChangeCount"
+            "value"     = "1"
+            "cooldown"  = "PT1M"
+          }
+        }
+        rule {
+          "metric_trigger" {
+            "metric_name"         = "Percentage CPU"
+            "metric_resource_uri" = "${azurerm_virtual_machine_scale_set.test.id}"
+            "time_grain"          = "PT1M"
+            "statistic"           = "Average"
+            "time_window"         = "PT5M"
+            "time_aggregation"    = "Average"
+            "operator"            = "LessThan"
+            "threshold"           = 25
+          }
+          "scale_action" {
+            "direction" = "Decrease"
+            "type"      = "ChangeCount"
+            "value"     = "1"
+            "cooldown"  = "PT1M"
+          }
+        }
+      }
+
+      notification {
+        "operation" = "Scale"
+        "email" {
+          "send_to_subscription_administrator"    = false
+          "send_to_subscription_co_administrator" = false
+        }
+      }
+    }`, rInt)
+}
+
+func testAccAzureRMAutoscaleSetting_recurrence(rInt int) string {
+	return fmt.Sprintf(`
+    variable "time_zone" {
+      type    = "string"
+      default = "Pacific Standard Time"
```

Comment:
Reflecting on this, maybe not, I was wondering if we could get away with the user inputting `UTC-8` rather than `Pacific Standard Time` - but that'd actually just complicate things